BugsAndExploits/LightSupression.md

```diff
@@ -1,22 +1,20 @@
 ---
-title: Light update suppression
+title: Light suppression
 description: Explanation and use of light suppression
 ---
 
-# light update suppression
+# Light Suppression
 
 Light update suppression is the delaying of light updates and then stopping the server to effectively delete them.
 
-## Technical explaination
-When a light update is produced via a block being moved, placed or broken it is placed in a queue for the light thread to process. Then when the light thread gets to the update it is processed and the light level is changed. How light update suppression works is it overloads that queue; delaying the processing of light updates. Then when the server is stopped the unprocessed light updates are never processed and the light level is wrong.
+## Technical explanation
+When a block being moved, placed, or broken a light update is produced. These light updates are placed in a queue for the light thread to process. Then when the light thread gets to that update it processes it and the light level is changed in the world. Light Suppression works by filling that queue with other light updates; delaying the processing of them. When the server is stopped any unprocessed light updates are deleted and thereby not processed, so the light level is wrong. 
 
 ## Use
-A light suppression machine is run. This typically consists of flashing redstone lamps or, in older light update suppressors, moving blocks. Light updates from light sources typically take longer to process.
-After aproximatly 20k light updates in the queue the machine is stopped. Normally this length of queue is around 3 seconds to process a light update. Any longer and the server will slow down to let the light thread catch up. 
-A light update that is to be delted is produced.
-The server is stopped either through the `/stop`
- command or crashing with an update suppressor. Results with `/stop`
- can be inconsistant so `/save-all flush` can be ran before to fix that. 
+1. A light suppression machine is run. This typically consists of flashing redstone lamps or, in older light update suppressors, moving blocks. Light updates from light sources typically take longer to process.
+2. After approximately 20k light updates are in the queue the machine is stopped. Normally the length of queue is around 3 seconds because if it were any longer the server will slow down to let the light thread catch up. 
+3. A light update that is to be deleted is produced.
+4. The server is stopped either through the `/stop` command or crashing with an update suppressor. Results with `/stop` can be inconsistent so `/save-all flush` can be ran before to fix that. 
```

Comment:
can someone verify point 2? does not sound correct to me.
point 3 is also phrased quite oddly

Reply:
I have experienced point 2 in 1.17+. I have no ideas how to phrase point three any better. Any ideas?